app/javascript/components/App.js

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchMessages } from './redux/app/app';
+
+const App = () => {
+  const state = useSelector((state) => state.messagesReducer);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchMessages());
+  }, []);
+  return (
+    <div>
+      Greetings:
+      {state.message}
+    </div>
```

Comment:
- [ ] Instead of directly showing the greeting message, I would recommend making the `Greeting` component and call from the `App` component as the `App` component is a parent one from where we would create its child to work on them and make the flow clear.

Reply:
Well addressed this